region/pom.xml

```diff
@@ -85,9 +91,10 @@
                             org.motechproject.nms.region.service;version=${project.version},
                             org.motechproject.nms.region.repository;version=${project.version},
                             org.motechproject.nms.region.domain;version=${project.version},
-                            org.motechproject.nms.region.exception;version=${project.version}
                         </Export-Package>
                         <Import-Package>
+                            org.motechproject.nms.csv.exception,

```

Comment:
Are these not explicitly referenced in the code?  I'm surprised they need to be added here.